docs/pages/Tests/index.js

```diff
@@ -14,6 +17,9 @@ export default class Tests extends Component {
       return (
          <Switch>
             <Route path={`${match.url}/oio-provider`} component={OIOProvider} />
+            <Route path={`${match.url}/text-color`} component={TextColor} />
```

Comment:
The other specimens for `Text` seem to be plural should `text-color` and `TextColor` be plural too?